<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneratorGroupBox</class>
 <widget class="QGroupBox" name="GeneratorGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generator</string>
  </property>
  <property name="title">
   <string>Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="comboBoxChannel">
     <item>
      <property name="text">
       <string>Channel 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Channel 2</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxChannelSettings">
     <property name="title">
      <string>&amp;Channel settings</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Signal form</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxForm">
          <item>
           <property name="text">
            <string>SINE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SQUARE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TRIANGLE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SAWU</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SAWD</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PWM</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Frequency, Hz</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBoxFreq">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000000</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Amplitude, V</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxAmpl">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Phase, Deg</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBoxPhase">
          <property name="maximum">
           <number>360</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Duty Cycle, %</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spinBoxDutyCycle">
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxChannelEnabled">
        <property name="text">
         <string>Enable output</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
